testnet/stacks-node/src/config.rs

```diff
@@ -105,13 +136,20 @@ mod tests {
 }
 
 impl ConfigFile {
-    pub fn from_path(path: &str) -> ConfigFile {
-        let content_str = fs::read_to_string(path).unwrap();
-        Self::from_str(&content_str)
+    pub fn from_path(path: &str) -> Result<ConfigFile, String> {
+        let content_str = fs::read_to_string(path);
+        if content_str.is_err() {
+            return Err(format!("Invalid path"));
+        }
+        Self::from_str(&content_str.unwrap())
     }
 
-    pub fn from_str(content: &str) -> ConfigFile {
-        let mut config: ConfigFile = toml::from_str(content).unwrap();
+    pub fn from_str(content: &str) -> Result<ConfigFile, String> {
+        let toml_result = toml::from_str(content);
```

Comment:
Stylistically, we try to avoid using `.unwrap()` or any of the `.expect*()` methods unless it can't be avoided.  Could you reformat this and other instances of this pattern to the following?

```
let config = match toml::from_str(content) {
   Ok(config) => config,
   Err(e) => {
      return Err(format!("Invalid toml: {}", &e));
   }
};
```